lib/cosmos/gui/qt.rb

```diff
@@ -446,7 +446,15 @@ def text=(value)
 end
 
 class Qt::PlainTextEdit
+  BLANK = ''.freeze
+  BREAK = '<br/>'.freeze
+  AMP = '&amp;'.freeze
+  NBSP = '&nbsp;'.freeze
+  GT = '&gt;'.freeze
+  LT = '&lt;'.freeze
   @@color_cache = {}
+  @@mapping = {'&'=>AMP,"\n"=>BLANK,"\s"=>NBSP,'>'=>GT,'<'=>LT}
+  @@regex = Regexp.union(@@mapping.keys)
```

Comment:
This is actually pretty cool: http://www.rubydoc.info/stdlib/core/Regexp#union-class_method